### PR TITLE
add "missing spacepen" poster

### PR DESCRIPTION
**Pull request by Hiucko**
## About the PR

added missing spacepen poster. poster sprite by @foboscheshir_ (<@740187230500421684>) on discord.

## Why / Balance

funny
## Media

image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Kadeo64** [comment]
the description should be changed to a live updating count of the amount of dislike reactions on the remove spacepens PR :trollface:

**Comment by VasilisThePikachu** [comment]
> the description should be changed to a live updating count of the amount of dislike reactions on the remove spacepens PR :trollface:

Fun fact! Its the second most disliked pr in this repo

**Comment by Kadeo64** [comment]
> Fun fact! Its the second most disliked pr in this repo

crazy how the top three most disliked PRs were the ones that were merged

(tbf seniors and slotbased were basically stepping stones for loadouts and gridinv but seniors still don't have their replacement)

File: Resources/Textures/Structures/Wallmounts/posters.rsi/meta.json [hunk 1]
```diff
@@ -1 +1 @@
...ster42_legit"}, {"name": "poster43_legit"}, {"name": "poster44_legit"}, {"name": "poster45_legit"}, {"name": "poster46_legit"}, {"name": "poster47_legit"}, {"name": "poster48_legit"}, {"name": "poster49_legit"}, {"name": "poster50_legit"}, {"name": "poster51_legit"}, {"name": "random_legit"}, {"name": "random_contraband"}, {"name": "random_anything"}, {"name": "poster52_contraband", "directions": 1, "delays": [[1.0]]}, {"name": "poster53_contraband", "directions": 1, "delays": [[1.0]]}, {"name": "poster54_contraband", "directions": 1, "delays": [[1.0]]}, {"name": "bagelmap", "directions": 1, "delays": [[1.0]]}, {"name": "deltamap", "directions": 1, "delays": [[1.0]]}, {"name": "marathonmap", "directions": 1, "delays": [[1.0]]}, {"name": "originmap", "delays": [[1.0]]}, {"name": "moosemap", "directions": 1, "delays": [[1.0]]}, {"name": "packedmap", "directions": 1, "delays": [[1.0]]}, {"name": "pillarmap", "directions": 1, "delays": [[1.0]]}, {"name": "salternmap", "directions": 1, "delays": [[1.0]]}, {"name": "splitmap", "directions": 1, "delays": [[1.0]]}, {"name": "lighthousemap", "directions": 1, "delays": [[1.0]]}, {"name": "waystationmap", "directions": 1, "delays": [[1.0]]}]}
```
Comment (TheShuEd) [inline Resources/Textures/Structures/Wallmounts/posters.rsi/meta.json:1]: bruh, please, can you fix formatting in this PR?

**Pull request merged**